examples/experimental/onnx/classification/onnx_ptq_classification.py

```diff
@@ -101,9 +97,11 @@ def run(onnx_model_path: str, output_model_path: str,
         "--batch_size", help="Batch size for initialization", type=int, default=1)
     parser.add_argument(
         "--shuffle", help="Whether to shuffle dataset for initialization", default=True)
-    parser.add_argument("--input_shape",
-        help="Model's input shape. e.g. [1, 3, 224, 224]. If it's not given, it automatically infers input shape.",
-                        nargs="+", type=int, default=None)
+    parser.add_argument(
+        "--input_shape", help="Model's input shape. e.g. [1, 3, 224, 224].",
+        nargs="+", type=int, default=None)
+    parser.add_argument(
+        "--input_keys", help="Model's input key.", type=str, default=None)
```

Comment:
What is an "input key"? Explanation is missing. Is "key" a proper word for this, something that has the expected meaning in the ONNX lexicon?

Reply:
Because `input_keys` have been used in sample code, `onnx_ptq_classification.py` and `onnx_ptq_segmentation.py`, I just used `input_keys`. But I checked that ONNX models use `name`, not `key`, and I changed every part that uses `input_keys` to `input_name`.